templates/events-header.php

```diff
@@ -39,20 +39,29 @@
 	<p class="event-sub-head">
 			<span class="event-host">
 				<?php
-				if ( 1 === count( $hosts ) ) :
-					esc_html_e( 'Host:', 'gp-translation-events' );
+				if ( count( $hosts ) > 0 ) :
+					if ( 1 === count( $hosts ) ) :
+						esc_html_e( 'Host:', 'gp-translation-events' );
+					else :
+						esc_html_e( 'Hosts:', 'gp-translation-events' );
+					endif;
 				else :
-					esc_html_e( 'Hosts:', 'gp-translation-events' );
+					esc_html_e( 'Created by:', 'gp-translation-events' );
+					?>
+					&nbsp;<a href="<?php echo esc_attr( get_author_posts_url( $user->ID ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $user->ID ) ); ?></a>
+					<?php
 				endif;
 				?>
 				<?php foreach ( $hosts as $host ) : ?>
-					<?php $user = get_userdata( $host->user_id() ); ?>
-					&nbsp;<a href="<?php echo esc_attr( get_author_posts_url( $user->ID ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $user->ID ) ); ?></a>
+					<?php $_user = get_userdata( $host->user_id() ); ?>
+					&nbsp;<a href="<?php echo esc_attr( get_author_posts_url( $_user->ID ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $_user->ID ) ); ?></a>
```

Comment:
I think the `$_user` variable is not needed

```suggestion
					&nbsp;<a href="<?php echo esc_attr( get_author_posts_url( $host->user_id() ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $host->user_id() ) ); ?></a>
```

Reply:
Good catch! Thank you.